index.html

```diff
@@ -138,16 +138,18 @@
     */
     
     // Working group info
-    wg:           "Accessible Rich Internet Applications Working Group",
-    wgURI:        "https://www.w3.org/WAI/ARIA/",
-    wgPublicList: "public-aria",
+	group: "aria",
```

Comment:
Nit: whitespace differences.

Reply:
not sure there is any consistency here in this part of the file on whitespace